src/modules/displayData.js

```diff
@@ -97,6 +105,12 @@ const createMovie = async () => {
       });
       fetchComment(movie.id);
     });
+    iconEl.addEventListener('click', () => {
+      postLike(`${movie.name}`);
+      let likeNum = Number(likeCount.textContent);
+      likeNum += 1;
+      likeCount.textContent = String(likeNum);
+    });
```

Comment:
- [Optional] Kindly make sure to create this counter function in a separate module according to [this requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/buisness_requirements.md#:~:text=Each%20counter%20should%20be%20implemented%20as%20a%20separate%20module.)